shared/js/background/atb.es6.js

```diff
@@ -120,13 +120,13 @@ var ATB = (() => {
                 var i = tabs.length, tab
                 while (i--) {
                     tab = tabs[i]
-                    
+
                     chrome.tabs.executeScript(tab.id, {
-                        file: 'content-scripts/on-install.js'
+                        file: 'public/js/content-scripts/on-install.js'
                     })
-                    
+
                     chrome.tabs.insertCSS(tab.id, {
-                        file: 'css/noatb.css'
+                        file: 'public/css/noatb.css'
```

Comment:
@jdorweiler as we discussed previously, this path has been incorrect for a while, but we haven't noticed it because the CSS file was injected correctly via the manifest.

I decided to fix the path instead of just removing these lines.